src/scaffold/entry_type.rs

```diff
@@ -101,6 +101,15 @@ pub fn scaffold_entry_type(
         }
     };
 
+    let widget_fields = fields
+        .iter()
+        .map(|f| f.clone())
+        .filter(|f| match f.widget {
+            Some(_) => true,
+            None => false
+        })
+        .collect();
```

Comment:
```suggestion
    let widget_fields = fields.clone()
        .iter()
        .filter(|f| match f.widget {
            Some(_) => true,
            None => false
        })
        .collect();
```
Less code, and I'm not 100% but I think it will also be marginally more efficient